client/matchmaking/matchmaking-disabled-card.tsx

```diff
@@ -112,7 +112,7 @@ export function ConnectedMatchmakingDisabledCard({
           <Headline6>{dateFormat.format(nextStartDate)}</Headline6>
           {nextEndDate && nextEndDate > nextStartDate ? (
             <>
-              <ToText>to</ToText>
+              <ToText>{t('matchmaking.disabledCard.startToEnd', 'to')}</ToText>
```

Comment:
generally I think this should be a `Trans` containing both dates and the bridge word, translating the bridge word on its own both doesn't give context and may not be a general enough solution to cover how date ranges are localized. Would also accept just replacing this with an en dash so it doesn't need to be translated